backend/src/api/tasks.js

```diff
@@ -1,20 +1,60 @@
-import { allQuery } from '../database/index.js'
-import { SELECT_USER_TASKS_QUERY } from '../database/queries.js'
+import { runQuery, allQuery } from '../database/index.js';
+import { SELECT_USER_TASKS_QUERY } from '../database/queries.js';
+
+export const createTask = async (req, res) => {

```

Comment:
I'm not get it for what 😊

Reply:
@gainullin81 this is no needed, tasks are created in `onStoreDocument` in utils.js – tasks shouldn't be created separately from cards.